packages/nextra-theme-docs/src/mdx-components.tsx

```diff
@@ -70,7 +70,10 @@ const createHeading = (
             : cn(
                 '_font-semibold _tracking-tight _text-slate-900 dark:_text-slate-100',
                 {
-                  h2: '_mt-10 _border-b _pb-1 _text-3xl _border-neutral-200/70 contrast-more:_border-neutral-400 dark:_border-primary-100/10 contrast-more:dark:_border-neutral-400',
+                  h2: cn(
+                    '_mt-10 _border-b _pb-1 _text-3xl _border-neutral-200/70 contrast-more:_border-neutral-400 dark:_border-primary-100/10 contrast-more:dark:_border-neutral-400',
+                    '[.nextra-steps_&]:_mt-8 [.nextra-steps_&]:_border-none [.nextra-steps_&]:_pb-0 [.nextra-steps_&]:_text-2xl'
+                  ),
```

Comment:
this changes is only for the docs theme, but in a blog h2 will be still bigger 
<img width="731" alt="image" src="https://github.com/user-attachments/assets/9aeded1d-56e8-4c9c-a557-95bae22fd545">

Reply:
Thank you for the suggestion. I hadn’t considered that the theme-blog could also use this component.

---

Additionally, I was thinking that the Steps component could support sub-steps, as image below:

![image](https://github.com/user-attachments/assets/9a7a602a-e2ec-4d8b-a016-27685366c488)

For better screen reader support for heading levels, MDX might look like:

```mdx
## Steps

<Steps>
### Foo
  <Steps>
    #### Foo step one
    #### Foo step two
  </Steps>
### Bar
  <Steps>
    #### Bar step one
    #### Bar step two
  </Steps>
### Baz
</Steps>
```

Should we consider supporting heading levels other than just `h1`? I’d like to hear your thoughts.

Reply:
> Should we consider supporting other heading levels? I’d like to hear your thoughts.

I think it would be nice and for below 👍

> For better screen reader support for heading levels